docs/api/puppeteer.page.waitforframe.md

```diff
@@ -32,6 +32,6 @@ Promise&lt;[Frame](./puppeteer.frame.md)&gt;
 
 ```ts
 const frame = await page.waitForFrame(async frame => {
-  return frame.name() === 'Test';
+  return (await frame.name()) === 'Test';
```

Comment:
frame.name() is a sync method and does not require an await

Reply:
yes, I was using `frame.title()`, but I see `name` method is sync, thanks